lib/server-starter.js

```diff
@@ -64,12 +64,14 @@ class Server extends EventEmitter {
    * @param {object} [options] - Optional settings
    * @param {boolean} [options.stdout=false] - Forward server output from STDOUT to STDOUT
    * @param {boolean} [options.stderr=true] - Forward server output from STDERR to STDERR
+   * @param {number} [options.win32Timeout=30000] - (Windows only) Max time to wait for server ready, in mS
```

Comment:
As far as i can see there is no reason at all for this to be Windows specific.

Reply:
It will have no effect on other platforms, at least how is it written.

If you mean to wait for connection ready at the TCP port in all cases, it could be done.

Reply:
The way it is done now makes little sense. Since the module doesn't become more portable, it just allows someone to use it in a Windows specific way. Instead of enabling one test to run on all platforms.